feature-libs/storefinder/components/store-finder-search/store-finder-search.component.html

```diff
@@ -27,25 +27,71 @@
         ></cx-icon>
       </div>
     </div>
+    <ng-template [ngTemplateOutlet]="searchActionButtons"></ng-template>
+  </div>
+
+  <div class="row align-items-end" *cxFeature="'a11yStoreFinderLabel'">
     <div class="col-md-12 col-lg-6">
-      <div class="row cx-search-links mb-3">
-        <div class="col-6">
-          <button
-            (click)="viewStoresWithMyLoc()"
-            class="btn btn-primary btn-block"
-          >
-            {{ 'storeFinder.useMyLocation' | cxTranslate }}
-          </button>
-        </div>
-        <div class="col-6">
-          <button
-            [routerLink]="['/store-finder/view-all']"
-            class="btn btn-primary btn-block"
-          >
-            {{ 'storeFinder.viewAllStores' | cxTranslate }}
-          </button>
-        </div>
+      <div class="form-group">
+        <label class="">
```

Comment:
Maybe including the label sth like this could help avoid the layout change? Just a thought 😉 

```
<label *cxFeature="'a11yStoreFinderLabel'" for="queryInput" class="text-left label-content">
  {{ 'storeFinder.findStoreBy' | cxTranslate }}
</label>
```

Reply:
@uroslates new label is fine. However one last fix needed here, as I'm sure auditors will point out the issue. Right now, JAWS still reads the aria-label in the field and ignores the visible label.
When we have a visible label, we need to programmatically associate it with the field and remove aria-label(s) since they override standard labels. In the case, we had an aria-label instead of a visible label, so we have to remove the aria-label. 
Therefore, can you please:
- Remove the aria-label from the field.
- Associate the visible label to the field.